DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@model SearchableDelegateViewModel
+@{
+  var activeTagCss = Model.IsActive ? "nhsuk-tag" : "nhsuk-tag nhsuk-tag--grey";
+  var selfRegTagCss = "nhsuk-tag nhsuk-tag--grey";
+  var adminTagCss = "nhsuk-tag nhsuk-tag--green";

```

Comment:
Given that this CSS doesn't have any logic, does it need to be a var or go directly on the HTML elements?

Reply:
Was just to make it easier to edit.